### PR TITLE
Instructions for setting up junction using Docker post junction upgrade.

### PR DESCRIPTION
**Pull request by RajatRajdeep**
**Major Changes:**

1. Added instructions in the ReadMe file for setting up junction using Docker post junction upgrade.
Task: https://github.com/pythonindia/junction/issues/753

2. Updated the psycopg version to resolve **SCRAM authentication requires libpq version 10 issue.**
Captured Junction Issue: https://github.com/pythonindia/junction/issues/759
The above issue occurred due to following psycopg issue which got resolved in version2.9.6: https://github.com/psycopg/psycopg2/issues/1360

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ Junction
...
+Prerequisites:
+1. Docker: You can download and install Docker from the official website at https://www.docker.com/get-started.
+
+Instructions:
+1. Copy the .env.sample file to a new .env file by running the following command: ```cp .env.sample .env```
+2. Create a local development settings file by running the following command: ```cp settings/dev.py.sample settings/dev.py```
+3. Build the junction_local image using the following command: ```docker build -t junction_local .```
+4. Start the project by running the following command: ```docker-compose up```
+5. Access the application at https://localhost:8888.
+
```
Comment (leshravnya) [inline README.md:20]: I think we need to add a note for the apple silicon machines as suggested below.
```suggestion
>Note:  
> For Apple Silicon machines set the following environment variable before building the junction image.
> ```sh
> export DOCKER_DEFAULT_PLATFORM=linux/amd64
> ```

```

**Review by leshravnya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RajatRajdeep) [reply README.md:20]: Thanks @leshravnya for the suggestion.
The  [issue](https://github.com/psycopg/psycopg2/issues/1360) exist in psycopg 2.9.5 which got resolved in version 2.9.6, So I have upgraded the psycopg version to 2.9.6.

Comment (leshravnya) [reply README.md:20]: Oh that's great then. Let me try it out.

**Review by leshravnya — APPROVED** [review]

**Comment by ananyo2012** [comment]
Thanks @RajatRajdeep for your contribution. This is merged to my feature branch for updating docs

**Pull request merged**